src/interaction/InteractionData.js

```diff
@@ -40,6 +40,93 @@ export default class InteractionData
+
+        /**
+         * Indicates whether or not the pointer device that created the event is the primary pointer.
+         * @see https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/isPrimary
+         * @type {Boolean}
+         */
+        this.isPrimary = false;
+        /**
+         * Indicates which button was pressed on the mouse or pointer device to trigger the event.
+         * @see https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button
+         * @type {number}
+         */
+        this.button = 0;
+        /**
+         * Indicates which buttons are pressed on the mouse or pointer device when the event is triggered.
+         * @see https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons
+         * @type {number}
+         */
+        this.buttons = 0;
+        /**
+         * The width of the pointer's contact along the x-axis, measured in CSS pixels.
+         * radiusX of TouchEvents will be represented by this value.
+         * @see https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/width
+         * @type {number}
+         */
+        this.width = 0;
+        /**
+         * The height of the pointer's contact along the y-axis, measured in CSS pixels.
+         * radiusY of TouchEvents will be represented by this value.
+         * @see https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/height
+         * @type {number}
+         */
+        this.height = 0;
+        /**
+         * The angle, in degrees, between the pointer device and the screen.
+         * @see https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/tiltX
+         * @type {number}
+         */
+        this.tiltX = 0;
+        /**
+         * The angle, in degrees, between the pointer device and the screen.
+         * @see https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/tiltY
+         * @type {number}
+         */
+        this.tiltY = 0;
+        /**
+         * The type of pointer that triggered the event.
+         * @see https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/pointerType
+         * @type {string}
+         */
+        this.pointerType = null;
+        /**
+         * Pressure applied by the pointing device during the event. A Touch's force property
+         * will be represented by this value.
+         * @see https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/pressure
+         * @type {number}
+         */
+        this.pressure = 0;
+        /**
+         * From TouchEvents (not PointerEvents triggered by touches), the rotationAngle of the Touch.
+         * @see https://developer.mozilla.org/en-US/docs/Web/API/Touch/rotationAngle
+         * @type {number}
+         */
+        this.rotationAngle = 0;
+        /**
+         * Twist of a stylus pointer.
+         * @see https://w3c.github.io/pointerevents/#pointerevent-interface
+         * @type {number}
+         */
+        this.twist = 0;
+        /**
```

Comment:
Very small stylistic thing, but could you add a new-line after the property assignment and before the nest Jsdoc block?